blueos_3_ppc64le_ib/spack.yaml

```diff
@@ -698,6 +724,8 @@ spack:
         prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-16.0.6
       - spec: spectrum-mpi@10.3.1.03rtm0%clang@=16.0.6.gcc.8.3.1
         prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-16.0.6-gcc-8.3.1
+      - spec: spectrum-mpi@10.3.1.03rtm0%clang@=16.0.6.cuda.11.8.0.gcc.11.2.1.cuda.11.8.0
```

Comment:
cuda.11.8.0 is in the spec twice. Is that intentional?

Reply:
Oops

Reply:
It’s me trying to choose between LC notations and ours.